woocommerce/woocommerce-ios
# Add translations for promo text

This PR updates the `download_metadata.swift` script to download the new `promo_text` field.
It also adds the translations that are currently available on GlotPress.

## To Test:
1. Run `download_metadata.swift`.
2. Verify that it works without errors and that the translations are downloaded.

Note that since translators are working on the current set, you may get some new strings when running the script. It's fine. The complete set of new strings will be committed when we'll prepare the final build for the current version, later this week.

Update release notes:
